jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -702,6 +702,13 @@ private InterpreterResult executeSql(String propertyKey, String sql,
               String results = getResults(resultSet,
                   !containsIgnoreCase(sqlToExecute, EXPLAIN_PREDICATE));
               interpreterResult.add(results);
+              try {
+                if (resultSet.isClosed()) {
+                  break;
+                }
+              } catch (Exception e) {
```

Comment:
I had to do this in a try-catch block because this method is not implemented in Hive https://github.com/apache/hive/blob/master/jdbc/src/java/org/apache/hive/jdbc/HiveBaseResultSet.java#L628-L630, which in my opinion is not an error.